### ISSUE TITLE
Navigating menus with arrow keys

### ISSUE THREAD

**Issue by Magnus26** [musescore/MuseScore#25128]
### Issue type

UX/Interaction bug (incorrect behaviour)

### Description with steps to reproduce

1. Press Alt+F to open the menu File.
2. Press arrow key Right four times to open the menu Add and submenu Notes.
3. Press Right again and see what happens: either the menu Edit is unexpectedly opened or the submenu closes.

### Supporting files, videos and screenshots

![Screenshot_20241011_083254](https://github.com/user-attachments/assets/df1c00bb-84ba-45bb-a010-a4ca671e7ac9)

### What is the latest version of MuseScore Studio where this issue is present?

4.4.2

### Regression

I was unable to check

### Operating system

Q4OS

### Checklist

- [X] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [X] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [X] I have attached all requested files and information to this report
- [X] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by Magnus26** [comment]
Now I can't get past (to the right) the menu Edit with arrow key Right. What is happening, really?

**Comment by Magnus26** [comment]
After some more testing it seems that the bug report only concerns MuseScore Studio with no score open. With a score open the menu navigation with arrow keys behaves OK.

**Comment by DmitryArefiev** [comment]
> After some more testing it seems that the bug report only concerns MuseScore Studio with **no score open**. With a score open the menu navigation with arrow keys behaves OK.

Yeah, confirm on my side. That happens only on **Home** tab or with opened score on **Publish** tab

Not a regression. But it works in other apps

I will edit the summary

**Comment by Magnus26** [comment]
> Yeah, confirm on my side. That happens only on **Home** tab or with opened score on **Publish** tab

It seems that the arrow key Right does not work when the submenu is greyed out.
